external/llvm-project/mlir/tools/mlir-shlib/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ if(MLIR_LINK_MLIR_DYLIB)
   set(INSTALL_WITH_TOOLCHAIN INSTALL_WITH_TOOLCHAIN)
 endif()
 
+if(MLIR_MIOPEN_SQLITE_ENABLED)
+  list(APPEND _DEPS "sqlite3")
+endif()
```

Comment:
`MLIR_MIOPEN_SQLITE_ENABLED` is set when MLIR need to read from perfdb. I'm not convinced this should be living in llvm directory or should be upstreamed at all.

Reply:
I don't know enough about this part of the build system to have an alternative in mind, but we should investigate this - thank you for noting the issue!